### PR TITLE
Make serverless platform label for fuse sidecar injection configurable

### LINKED ISSUE #1930: [BUG] Make serverless platform label configurable for fuse sidecar injection
**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

### PR DESCRIPTION
**Pull request by TrafalgarZZZ**
### Ⅰ. Describe what this PR does
Make serverless platform label for fuse sidecar injection configurable

### Ⅱ. Does this pull request fix one issue?

fixes #1930 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: charts/fluid/fluid/templates/webhook/webhook.yaml [hunk 1]
```diff
@@ -32,6 +32,16 @@ spec:
                   fieldPath: metadata.namespace
             - name: TIME_TRACK
               value: "false"
+            {{- if .Values.webhook.serverlessInject }}
+            {{- if .Values.webhook.serverlessInject.platformLabel }}
```
Comment (cheyang) [inline charts/fluid/fluid/templates/webhook/webhook.yaml:36]: How about naming it `platformKey` and `KEY_SERVERELESS_PLATFORM` ? Just a suggestion. WDYT?

File: pkg/utils/annotations.go [hunk 2]
```diff
@@ -15,18 +15,38 @@ limitations under the License.
...
+	}
+	if envVal, exists := os.LookupEnv(common.EnvServerlessPlatformVal); exists {
+		ServerlessPlatformVal = envVal
+		stdlog.Printf("Found %s value %s, using it as ServerlessPlatformLabelValue", common.EnvServerlessPlatformVal, envVal)
+	}
+}
 
 func ServerlessEnabled(infos map[string]string) (match bool) {
-	return matchedValue(infos, common.ServerlessPlatform, common.ASKPlatform) || enabled(infos, common.InjectServerless) || enabled(infos, common.InjectFuseSidecar)
+	return matchedValue(infos, ServerlessPlatformKey, ServerlessPlatformVal) || enabled(infos, common.InjectServerless) || enabled(infos, common.InjectFuseSidecar)
```
Comment (cheyang) [inline pkg/utils/annotations.go:41]: What will happen if the ServerlessPlatformKey and ServerlessPlatformVal is not set?

File: charts/fluid/fluid/templates/webhook/webhook.yaml (hunk 1 above)
Comment (TrafalgarZZZ) [reply charts/fluid/fluid/templates/webhook/webhook.yaml]: Done

File: pkg/utils/annotations.go (hunk 2 above)
Comment (TrafalgarZZZ) [reply pkg/utils/annotations.go]: Have add an extra check empty string process. If not set, it returns false.

**Review by cheyang — APPROVED** [review]

**Pull request merged**